sdk/attestation/azure-security-attestation/samples/README.md

```diff
@@ -0,0 +1,176 @@
+
+Examples of shared instances are:
+
+* sharedeus2.eus2.attest.azure.net
+* sharedcae.cae.attest.azure.net
+* shareduks.uks.attest.azure.net
+
+### AAD Mode
+
+AAD mode instances are intended for customers who trust ARM RBAC for authorization
+decisions related to policy management. Attestation policies are allowed to be
+either be signed or unsigned.
+
+### Isolated Mode
+
+Isolated mode instances are intended for customers who desire an additional level
+of authorization beyond that which is allowed by ARM RBAC authorization. When a
+customer creates an isolated attestation instance, they also need to create an
+RSA asymmetric key pair and an X.509 certificate which contains that asymmetric
+key (the certificate can be self signed, or it can be issued by a certificate
+authority). Attestation policies MUST be signed with one of the private keys
+associated with the instance (either at instance creation or added with the [add_policy_management_certificate][add_policy_management_cert] API.
+
+## Sample Requirements
+
+These samples are written with the assumption that the following environment
+variables have been set by the user:
+
+* ATTESTATION_AAD_URL - the base URL for an attestation service instance in AAD mode.
+* ATTESTATION_ISOLATED_URL - the base URL for an attestation service instance in Isolated mode.
+* ATTESTATION_LOCATION_SHORT_NAME - the short name for the region in which the
+    sample should be run - used to interact with the shared endpoint for that
+    region.
+* ATTESTATION_ISOLATED_SIGNING_CERTIFICATE - The DER encoded form of the signing
+    certificate used to create an isolated attestation instance, Base64 encoded.
+* ATTESTATION_ISOLATED_SIGNING_KEY - The DER encoded of an RSA Private key, which
+    was used to create an isolated attestation service instance.
+
+The tests also assume that the currently logged on user is authorized to call
+into the attestation service instance because they use [DefaultAzureCredential](https://docs.microsoft.com/python/api/azure-identity/azure.identity.defaultazurecredential?view=azure-python) for authorization.
+
+## Samples descriptions
+
+**File Name** | **Description**
+|-----|-------|
+| [sample_authentication.py](https://aka.ms/azsdk/guideline/links) and [sample_authentication_async.py](https://aka.ms/azsdk/guideline/links) | Authenticate a connection with the attestation service.|
+| [sample_attest_enclave.py](https://aka.ms/azsdk/guideline/links) and [sample_attest_enclave_async.py](https://aka.ms/azsdk/guideline/links) | Attest an SGX and OpenEnclave enclave with the attestation service. Also shows how to use the TokenValidationOptions to perform additional validation of the returned attestation token and validation keys.|
+|[sample_get_set_policy.py](https://aka.ms/azsdk/guideline/links) and [sample_get_set_policy_async.py](https://aka.ms/azsdk/guideline/links) | Policy manipulation operations - Get, Set, Reset on different attestation types.|
```

Comment:
Reminder to change these links after merging this PR. You can include the links as they would appear and we can override check-enforcer when that's the only failure.